README.md

```diff
@@ -110,6 +110,58 @@ This SDK abides by our Client SDK Deprecation Policy. For more information on th
 
 Xcode 15.0+ is required to run the demo app.
 
+## Payment Buttons
+
+The Braintree iOS SDK now allows merchants to draw and render both PayPal and Venmo payment buttons using a discrete set of parameters. 
+
+*Note:* Ensure you include the `BraintreeUIComponents` module in your project to use this feature.
+
+If you would like to draw a Venmo branded button in your mobile app, you can do so like this:
+
+```
```

Comment:
If we change this here and below it will show as swift highlighted syntax:

```suggestion
```swift
```